### PR TITLE
Updates for 0.14

### PR DESCRIPTION
**Pull request by kritzcreek**

### CODE REVIEW

**Review by natefaubion — APPROVED** [review]

**Comment by kritzcreek** [comment]
I'll release this as v8.0.0

File: src/Psa/Output.purs [hunk 1]
```diff
@@ -234,7 +234,7 @@ trimPosition lines pos =
       Just _            -> trimComment col l
       _                 -> Nothing
 
-  -- TODO: this breaks if "--" is insidentitye a quoted string.
```
Comment (nwolverson) [inline src/Psa/Output.purs:237]: :smile:

Comment (natefaubion) [reply src/Psa/Output.purs:237]: Some VIM vomit there probably.

FYI all this range trimming stuff is totally unnecessary now.

Comment (nwolverson) [reply src/Psa/Output.purs:237]: Agreed. If there isn't a version support policy I'd propose my IDE one which is "current and previous major version". In any case removing this wouldn't completely break against older versions - it just gives looser ranges, right

Comment (natefaubion) [reply src/Psa/Output.purs:237]: The ranges thing was fixed in 0.13, prior to that, yeah, the ranges included whitespace and comments.

**Pull request merged**